### PR TITLE
AWS Access Request Template Refinement

### PR DESCRIPTION
**Pull request by td-usds**
This PR updates the AWS Access Request issue template. The wording was ambiguous and not totally clear in requesting information about the target principal (i.e. the principal for whom access is being requested). Third party filers can be mistaken in thinking that the requested information is about them (i.e. the filer) and not the target principal.
This PR addresses this confusion by making it clear that the requested information should cover the target principal.

This PR also adds an extra field '`additional-notes`' with instructions to use this to specify a VOR that is not listed in the dropdown, as well as to use this to justify the requested access.

### CODE REVIEW

**Review by iriteshp — COMMENTED** [review]
Looks good to me.

**Review by calquisola — APPROVED** [review]
looks good to me.

File: .github/ISSUE_TEMPLATE/aws-access-request.yml [hunk 1]
```diff
@@ -41,59 +41,65 @@ body:
         - Magnum Opus - Paula G. Mendoza
         - Liberty IT -  Michael G. Lovejoy
         - Accenture Federal Services - Alexa Elliot
-        - Other
+        - Other - please specify in 'Additional Notes'
```
Comment (rmtolmach) [inline .github/ISSUE_TEMPLATE/aws-access-request.yml:44]: I like this 👍

File: .github/ISSUE_TEMPLATE/aws-access-request.yml [hunk 2]
```diff
@@ -41,59 +41,65 @@ body:
...
-      description: The requested access will be revoked at the start of this date. Enter a date (and time if applicable) or 'ongoing' for non-expiring access.
+      description: The requested access will be revoked at the start of this date. Enter a date (and time if applicable) or 'ongoing' for non-expiring access. If ongoing is selected, add a justification for this to the 'Additional Notes' field.
       placeholder: timestamp
     validations:
       required: true
+  - type: textarea
+    id: additional-notes
+    attributes:
+      label: Additional Notes
+      description: Use this section to add notes, such as justification for the access request, COR information when 'Other' was selected, etc.
```
Comment (rmtolmach) [inline .github/ISSUE_TEMPLATE/aws-access-request.yml:101]: Change "COR" to "VOR"? Unless we want them to add additional info for both.

**Review by rmtolmach — COMMENTED** [review]
I like these changes. 

I have it on my list to add some documentation to the Platform website that explains the whole, "you must be in a roster" thing. But until then, I like what you have here.

File: .github/ISSUE_TEMPLATE/aws-access-request.yml [hunk 3]
```diff
@@ -7,17 +7,17 @@ body:
...
         :wave: **_Instructions_**
-        1. Before being granted access, you will need to either be listed on the [Platform Roster](https://vfs.atlassian.net/wiki/spaces/AP/pages/1908834623/Platform+Roster) or have started your [Platform Orientation](https://depo-platform-documentation.scrollhelp.site/getting-started/Platform-Orientation.1877344532.html) and be listed on the [VFS Roster](https://docs.google.com/spreadsheets/d/11dpCJjhs007uC6CWJI6djy3OAvjB8rHB65m0Yj8HXIw/edit?folder=0ALlyxurHpUilUk9PVA#gid=2042046665)
-        2. Change the **Title** to include your name
-        3. Add the label used by your team (eg: `BAH-526`)
+        1. Before being granted access, the principal for whom access is requested (the 'target principal') must either be listed on the [Platform Roster](https://vfs.atlassian.net/wiki/spaces/AP/pages/1908834623/Platform+Roster) or have started your [Platform Orientation](https://depo-platform-documentation.scrollhelp.site/getting-started/Platform-Orientation.1877344532.html) and be listed on the [VFS Roster](https://docs.google.com/spreadsheets/d/11dpCJjhs007uC6CWJI6djy3OAvjB8rHB65m0Yj8HXIw/edit?folder=0ALlyxurHpUilUk9PVA#gid=2042046665)
```
[the remark this answers is not in the document]
Comment (td-usds) [reply .github/ISSUE_TEMPLATE/aws-access-request.yml]: fixed in af0b9662029143323dcd325b078fcaf9e623ee95

File: .github/ISSUE_TEMPLATE/aws-access-request.yml (hunk 1 above)
Comment (td-usds) [reply .github/ISSUE_TEMPLATE/aws-access-request.yml:44]: Like it: fixed in 208fa6b4843f130b49fff88d32d6165c58e67445

File: .github/ISSUE_TEMPLATE/aws-access-request.yml (hunk 2 above)
Comment (td-usds) [reply .github/ISSUE_TEMPLATE/aws-access-request.yml]: Good catch, will be fixed via 3063b55bdebcfc27a81ab74478ff295b0a3ec2e3

**Review by rmtolmach — APPROVED** [review]

**Pull request merged**